bootstrap/pkg/kfapp/gcp/gcp.go

```diff
@@ -637,65 +590,6 @@ func (gcp *Gcp) generateDMConfigs() error {
 	return nil
 }
 
-func (gcp *Gcp) downloadK8sManifests() error {
-	appDir := gcp.Spec.AppDir
-	k8sSpecsDir := path.Join(appDir, K8S_SPECS)
-	k8sSpecsDirErr := os.MkdirAll(k8sSpecsDir, os.ModePerm)
-	if k8sSpecsDirErr != nil {
-		return fmt.Errorf("cannot create directory %v Error %v", k8sSpecsDir, k8sSpecsDirErr)
-	}
-	daemonsetPreloaded := filepath.Join(k8sSpecsDir, "daemonset-preloaded.yaml")
-	url := "https://raw.githubusercontent.com/GoogleCloudPlatform/container-engine-accelerators/stable/nvidia-driver-installer/cos/daemonset-preloaded.yaml"
-	urlErr := gogetter.GetFile(daemonsetPreloaded, url)
-	if urlErr != nil {
-		return fmt.Errorf("couldn't download %v Error %v", url, urlErr)
-	}
-	rbacSetup := filepath.Join(k8sSpecsDir, "rbac-setup.yaml")
```

Comment:
Where does stackdriver part go?

Reply:
It's not needed if we apply gcp/prometheus.
Will add in separate PR.